bundles/org.openhab.binding.ferroamp/README.md

```diff
@@ -0,0 +1,203 @@
+| ssos3-ts             | String                     | R          | Time stamp when message was published    |
+| ssos3-udc            | Number:ElectricPotential   | R          | DC link voltage as measured by SSO-3     |
+| ssos3-faultcode      | String                     | R          | See section 4.1.3.1                      |
+| ssos3-ipv            | Number:ElectricCurrent     | R          | Measured on PV string side               |
+| ssos3-upv            | Number:ElectricPotential   | R          | Measured on PV string side               |
+| ssos3-id             | String                     | R          | Unique identifier of SSO-3               |
+| eso-faultcode        | String                     | R          | See section 4.1.3.1                      |
+| eso-id               | String                     | R          | Unique identifier                        |
+| eso-ibat             | Number:ElectricCurrent     | R          | Measured on battery side                 |
+| eso-ubat             | Number:ElectricPotential   | R          | Measured on battery side                 |
+| eso-relaystatus      | String                     | R          | 0 = relay closed (i.e running power), 1 = relay open/disconnected, 2 = precharge|
+| eso-soc              | Number:Dimensionless       | R          | State of Charge for ESO                  |
+| eso-temp             | Number:Temperature         | R          | Measured inside ESO                      |
+| eso-wpbatcons        | Number:Energy              | R          | Total energy produced by ESO, i.e total energy charged|
+| eso-wbatprod         | Number:Energy              | R          | Total energy produced by ESO, i.e total energy discharged|
+| eso-udc              | Number:Power               | R          | DC link voltage as measured by ESO       |
+| eso-ts               | String                     | R          | Time stamp when message was published    |
+| esm-soh              | Number:Dimensionless       | R          |                                          |
+| esm-soc              | Number:Dimensionless       | R          |                                          |
+| esm-rated-capacity   | Number:Energy              | R          | Rated capacity of battery                |
+| esm-id               | String                     | R          | Unique identifier of battery. If available, this will be the unique id that the battery reports. |
+| esm-rated-power      | Number:Power               | R          | Rated power of battery                   |
+| esm-status           | String                     | R          | Dependent on battery manufacturer        |
+| esm-ts               | String                     | R          | Time stamp when message was published    |
+
+The following channels are available for `ferroamp` energy system configuration. Please, see Ferroamp documentation for more details.
```

Comment:
These channels are just like any other, they are just write channels. Maybe add them to the table aboce? Also are they write only or R/W ?